my-fund-backend/src/main/java/com/myfund/controller/BudgetController.java

```diff
@@ -0,0 +1,28 @@
+package com.myfund.controller;
+
+import com.myfund.model.DTO.BudgetDTO;
+import com.myfund.model.DTO.CreateBudgetDTO;
+import com.myfund.model.User;
+import com.myfund.service.BudgetService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin(origins = "http://localhost:4200")
```

Comment:
Probably for later, but this values should come from some config

Reply:
I moved this value to the application properties.